en/advanced_config/filter_tuning.md

```diff
@@ -0,0 +1,80 @@
+# Filter Tuning & Control Latency
+
+Filters can be used to trade off [control latency](#control-latency), which affects flight performance, and noise filtering, which impacts both flight performance and motor health.
+
+This topic provides an overview of control latency and PX4 filter tuning.
+
+## Control Latency
+
+The *control latency* is the delay from a physical disturbance of the vehicle until the motors react to the change.
+
+:::tip
+Lowering latency allows you to increase the rate **P** gains, which results in better flight performance.
+Even one millisecond difference in the latency can have a significant impact.
+:::
+
+The following factors affect control latency:
+- A soft airframe or soft vibration mounting increases latency (they act as a filter).
+- Low-pass filters in software and on the sensor chip trade off increased latency for improved noise filtering.
+- PX4 software internals: the sensor signals need to be read in the driver and then pass through the controller to the output driver.
+- The IO chip (MAIN pins) adds about 5.4 ms latency compared to using the AUX pins (this does not apply to a *Pixracer* or *Omnibus F4*, but does apply to a Pixhawk).
+  To avoid the IO delay, disable [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and attach the motors to the AUX pins instead.
+- PWM output signal: enable One-Shot to reduce latency ([PWM_RATE](../advanced_config/parameter_reference.md#PWM_RATE)=0).
+
+Below we look at the impact of the low pass filters.
+
+## Filters
+
+This is the filtering pipeline for the controllers in PX4:
+- On-chip DLPF for the gyro sensor.
+  The cutoff frequency is set to 98Hz and it is sampled at 1kHz.
+- Low-pass filter on the gyro sensor data.
+  It can be configured with the [IMU_GYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_GYRO_CUTOFF) parameter.
+- A separate low-pass filter on the D-term.
+  The D-term is most susceptible to noise while slightly increased latency does not negatively affect performance.
+  For this reason the D-term has a separately-configurable low-pass filter, [IMU_DGYRO_CUTOFF](../advanced_config/parameter_reference.md#IMU_DGYRO_CUTOFF).
+- A slewrate filter on the motor outputs ([MOT_SLEW_MAX](../advanced_config/parameter_reference.md#MOT_SLEW_MAX)).
+  Generally not used.
+
+To reduce the control latency, we want to increase the cutoff frequency for the low-pass filters.
+However this is a trade-off as it will also increase the noise of the signal that is fed to the motors.
+Noise on the motors has the following consequences:
+- Motors and ESCs can get hot, to the point where they get damaged.
+- Reduced flight time because the motors continuously change their speed.
+- Visible random small twitches.
+
+The best filter settings depend on the vehicle.
+The defaults are set conservatively — such that they work on lower-quality setups as well.
+
+## Filter Tuning
```

Comment:
We could mention here as well that too high gains (particularly D) can show up as vibrations

Reply:
@bkueng Not sure best place for this, so I updated the warning in the 2nd paragraph below to include this as part of "don't use filter tuning to correct for vibration" Work for you?

![image](https://user-images.githubusercontent.com/5368500/114828864-c0b84f80-9e0d-11eb-9f13-e1b642f1939f.png)